### PR TITLE
Add numbers with start 4

### PR DESCRIPTION
**Pull request by emtudo**

### CODE REVIEW

**Comment by fzaninotto** [comment]
Any pointer to the accepted formats? I'd like to avoid changing formatters too often, so I want to make sure this is completely compliant to the norm.

**Comment by emtudo** [comment]
In Brazil there are fixed numbers starting with 2, 3 and 4.

**Comment by fzaninotto** [comment]
I understand that, but I'd like you to add a link in the code as a phpDoc, pointing to an official description of mobile phone formats. That way, any developer in the future can know when the format changes in the norm, and update Faker accordingly.

**Comment by emtudo** [comment]
Since december 2016 all mobile phone numbers in brazil begin with 9 and landlines 2, 3 or 4.
I do not believe it will be necessary to change anything in the near future.

**Comment by emtudo** [comment]
http://www.anatel.gov.br/Portal/exibirPortalPaginaEspecial.do?org.apache.struts.taglib.html.TOKEN=9594e1d11fbc996d52bda44e608bb744&codItemCanal=1794&pastaSelecionada=2984

**Comment by fzaninotto** [comment]
> add a link in the code as a phpDoc

**Comment by emtudo** [comment]
done!

File: src/Faker/Provider/pt_BR/PhoneNumber.php [hunk 1]
```diff
@@ -5,8 +5,12 @@
 class PhoneNumber extends \Faker\Provider\PhoneNumber
 {
 
-    protected static $landlineFormats = array('2###-####', '3###-####');
+    protected static $landlineFormats = array('2###-####', '3###-####', '4###-####');
 
+    /**
+    * Since december 2016 all mobile phone numbers in brazil begin with 9 and landlines 2, 3 or 4.
+    * numbershttp://www.anatel.gov.br/Portal/exibirPortalPaginaEspecial.do?org.apache.struts.taglib.html.TOKEN=9594e1d11fbc996d52bda44e608bb744&codItemCanal=1794&pastaSelecionada=2984
```
Comment (fzaninotto) [inline src/Faker/Provider/pt_BR/PhoneNumber.php]: s/numbershttp/@link http/

**Comment by emtudo** [comment]
Sorry!
Done!

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**